src/partials/benefits/benefits.html

```diff
@@ -1,6 +1,6 @@
 <section class="benefits">
   <div class="container">
-    <div class="benefits-wrapper">
+    <div class="section-pb">
       <h2 class="benefits-section-title">BENEFITS OF WORKING WITH ME</h2>
```

Comment:
You can use 'section-title' class to h2 from base.css file. There is no need to duplicate styles that already exist in base.css